python/paddle/v2/fluid/layers/control_flow.py

```diff
@@ -687,11 +687,10 @@ def topk(input, k):
 
 
 def lod_tensor_to_array(x, table):
-    """This function performs the operation that converts an LOD_Tensor to
-       an array.
+    """ Convert a LOD_TENSOR_ARRAY to an TensorArray.
```

Comment:
Should it be `LOD_TENSOR_ARRAY` or `LoDTensor` here

Reply:
Done. thank you.